src/main/java/uk/gov/hmcts/reform/profilesync/service/impl/ProfileSyncServiceImpl.java

```diff
@@ -95,49 +95,47 @@ public String getS2sToken() {
     }
 
 
-    public List<IdamClient.User> getSyncFeed(String bearerToken, String searchQuery) throws UserProfileSyncException {
+    public Set<IdamClient.User> getSyncFeed(String bearerToken, String searchQuery)throws UserProfileSyncException {
         Map<String, String> formParams = new HashMap<>();
         formParams.put("query", searchQuery);
 
-        List<IdamClient.User> updatedUserList = new ArrayList<>();
+        Set<IdamClient.User> updatedUserList = new HashSet<>();
```

Comment:
updatedUserList is a set and not a list

Reply:
yes modified from list to set

Reply:
the name should be changed to updatedUserSet